wgengine/magicsock/magicsock_linux.go

```diff
@@ -38,8 +38,15 @@ const (
 	discoMinHeaderSize = len(disco.Magic) + 32 /* key length */ + disco.NonceLen
 )
 
-// Enable/disable using raw sockets to receive disco traffic.
-var debugDisableRawDisco = envknob.RegisterBool("TS_DEBUG_DISABLE_RAW_DISCO")
+var (
+	// Enable/disable using raw sockets to receive disco traffic.
+	debugDisableRawDisco = envknob.RegisterBool("TS_DEBUG_DISABLE_RAW_DISCO")
```

Comment:
we could get away with just one knob (enable), though depends on how you picture this evolving

Reply:
Did this; thanks